File: program/program/convert-string-to-consonantcase/convert_string_to_consonantcase.hs
import Data.Char (toLower, toUpper)

consonantCase :: String -> String
consonantCase [] = []
consonantCase (x:xs)
    | elem x "aeiouAEIOU" = toLower x : consonantCase xs
    | otherwise = toUpper x : consonantCase xs

main :: IO ()
main = do
    putStrLn "Enter a string:"
    input <- getLine
    putStrLn $ "Consonant-case: " ++ consonantCase input

File: program/program/convert-string-to-consonantcase/convertStringToConsonantcase.scala
object ConsonantCaseConverter {
  def isVowel(c: Char): Boolean = {
    "AEIOUaeiou".contains(c)
  }

  def consonantCase(input: String): String = {
    input.map {
      case c if !c.isLetter => c
      case c if isVowel(c) => c.toLower
      case c => c.toUpper
    }
  }

  def main(args: Array[String]): Unit = {
    println("Enter a string: ")
    val inputString = scala.io.StdIn.readLine()
    val convertedString = consonantCase(inputString)
    println("Consonant Case String: " + convertedString)
  }
}
